score/scripts/download_cbg.py

```diff
@@ -0,0 +1,106 @@
+import csv
+import requests
+import zipfile
+import os
+import json
+
+with requests.Session() as s:
+    # the fips_states_2010.csv is generated from data here
+    # https://www.census.gov/geographies/reference-files/time-series/geo/tallies.html
+    with open("data/fips_states_2010.csv") as csv_file:
```

Comment:
you can use `pathlib` to work with paths in an OS-, and root-directory independent way - more [here](https://realpython.com/python-pathlib/) but its a lifesaver

Reply:
Additionally you can use importlib.resources (more [here](https://realpython.com/python-import/#introducing-importlibresources)) to treat the filesystem like a package, allowing you to run the script from any location (including later direct from within a docker container)

Reply:
Addressed in latest commit of PR